client/src/components/Home.tsx

```diff
@@ -14,9 +14,14 @@ function Home() {
       <hr />
       <ProjectSelect />
       <hr />
-      <button disabled={totalProcessingCount() > 0} onclick={() => totalProcessingCount() == 0 ? commands.makeAllDats() : undefined}>
-        Make all DATs
-      </button>
+      <div class="flex flex-row space-x-5">
+        <button disabled={totalProcessingCount() > 0} onclick={() => totalProcessingCount() == 0 ? commands.makeAllYamls() : undefined}>
+          Make all YAMLs
```

Comment:
So far I've gone with the nomenclature of "Export from DATs" instead of "Make YAMLs" on the front-end, but have kept it "make yaml" in the backend for now. Will probably change it to "export DATs" on the backend too, or something else entirely on both ends if a better name for it comes to mind.